plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/CachingJdbcClient.java

```diff
@@ -578,6 +578,12 @@ public OptionalInt getMaxWriteParallelism(ConnectorSession session)
         return delegate.getMaxWriteParallelism(session);
     }
 
+    @Override
```

Comment:
it helps make the next commit's diff much cleaner - no need to pay attention to the interface change when you can just focus on the implementation details which are the ones which need to be actually reviewed.

As before, there are no "hard rules, only guidelines". In this case practically people can not revert this commit without any impact since it's a no-op so it doesn't matter.